CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ release.
   <th title="LTS Until 2025-04"><a href="doc/changelogs/CHANGELOG_V18.md">18</a> (LTS)</th>
 </tr>
 <tr>
+  <td valign="top">
 <b><a href="doc/changelogs/CHANGELOG_V23.md#23.2.0">23.2.0</a></b><br/>
```

Comment:
After https://github.com/nodejs/node/commit/cb7d855f88d4caad63d6563bb78e2772382b4c0b, there's a merge conflict on `main`
```suggestion
<b><a href="doc/changelogs/CHANGELOG_V23.md#23.2.0">23.3.0</a></b><br/>
<a href="doc/changelogs/CHANGELOG_V23.md#23.2.0">23.2.0</a><br/>
```

Reply:
Rebased and fixed conflicts.